game-core/src/main/java/games/strategy/triplea/ui/TooltipProperties.java

```diff
@@ -46,11 +46,21 @@ public static TooltipProperties getInstance() {
     return ttp;
   }
 
+  public String getAppendedToolTip(final UnitType ut, final PlayerID playerId) {
+    return getToolTip(ut, playerId, true);
+  }
+
   public String getToolTip(final UnitType ut, final PlayerID playerId) {
+    return getToolTip(ut, playerId, false);
+  }
+
+  private String getToolTip(final UnitType ut, final PlayerID playerId, final boolean isAppending) {
+    final String append = isAppending ? ".append" : "";
```

Comment:
Minor, you could avoid the conditional here potentially by passing the append value directly, eg:
```
 return getToolTip(ut, playerId, ".append");
 return getToolTip(ut, playerId, "");
```
```
private String getToolTip(final UnitType ut, final PlayerID playerId, final String appendString) {
```

Reply:
Considered it but figured pushing the append value as far down as possible was better.